Vienna.xcodeproj/xcshareddata/xcschemes/Vienna.xcscheme

```diff
@@ -40,7 +40,7 @@
       buildConfiguration = "Development"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
-      shouldUseLaunchSchemeArgsEnv = "YES"
+      shouldUseLaunchSchemeArgsEnv = "NO"
```

Comment:
If I correctly understand your intent, you probably should leave this to "YES"

Reply:
Setting this to `NO` is necessary to change the launch arguments. Otherwise you can’t edit the field.